files/default/sql/ddl/updates/undo/2.4.0__undo.sql

```diff
@@ -74,3 +74,5 @@ ALTER TABLE `hopsworks`.`jupyter_settings` DROP COLUMN `no_limit`;
 ALTER TABLE `hopsworks`.`oauth_login_state` MODIFY COLUMN `state` VARCHAR(128);
 
 ALTER TABLE `hopsworks`.`feature_group` DROP COLUMN `event_time`;
+
+DROP VIEW project_username;
```

Comment:
is this needed? if yes can we move it to 2.6.0

Reply:
had removed just before you saw this.